src/Phinx/Db/Adapter/AdapterInterface.php

```diff
@@ -156,6 +156,16 @@ public function setOutput(OutputInterface $output);
      */
     public function getOutput();
 
+    /**
+     * Returns a new Phinx\Db\Table\Column using the existent data domain.
+     *
+     * @param string $columnName The desired column name
+     * @param string $type The type for the column. Can be a data domain type.
+     * @param array $options Options array
+     * @return \Phinx\Db\Table\Column
+     */
+    public function getColumnForType($columnName, $type, $options);
```

Comment:
array typehint on options?

Reply:
I think this is no bc break because it wasn't implemented before and the code is basically the same as when it was part of the original method if-else block.

However I can see it can be `protected` method anyway, since it is not intended to be called from public user side (only from concrete adapter classes).